modules/core/src/test/java/org/locationtech/jts/algorithm/CentroidTest.java

```diff
@@ -0,0 +1,50 @@
+package org.locationtech.jts.algorithm;
+
+import junit.framework.TestCase;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.WKTReader;
+
+public class CentroidTest extends TestCase {
+
+    private static final double tolerance = 1e-10;
```

Comment:
A minor style point - I like to make static final fieldnames uppercase, so it's obvious that they are constants.